diff --git a/src/opend/dspg/cmbs/test/appcmbs/appcall.c b/sw_package_host/test/appcmbs/appcall.c
index cbfe4f9..6703745 100644
--- a/src/opend/dspg/cmbs/test/appcmbs/appcall.c
+++ b/sw_package_host/test/appcmbs/appcall.c
@@ -135,17 +135,7 @@ extern u8 g_HoldCfm;
 extern u8 g_TransferAutoCfm;
 extern u8 g_ConfAutoCfm;
 extern u8 g_EarlyMediaAutoCfm;
-extern u8 u8_callId;
 
-static hsSuccessfullyConnected successfullyConnected;
-static hsSuccessfullyReleased successfullyReleased;
-
-void initMsgParserCall(hsSuccessfullyConnected connectedClb, hsSuccessfullyReleased releasedClb)
-{
-  if(!connectedClb || !releasedClb) return;
-  successfullyConnected = connectedClb;
-  successfullyReleased = releasedClb;
-}
 
 #define CALL_INSTANCE_UNUSED 0 // in principle 0 is a legal call instance, but it is only theoretical. it will be used only after 4Giga calls... we will live with this.
 #define MAX_RELEASE_CALL_INST 2
@@ -4946,12 +4936,10 @@ int app_CallEntity(void *pv_AppRef, E_CMBS_EVENT_ID e_EventID, void *pv_EventDat
         return TRUE;
 
     case CMBS_EV_DEE_CALL_RELEASE:
-        if(successfullyReleased != 0) successfullyReleased();
         app_OnCallRelease(pv_AppRef, pv_EventData);
         return TRUE;
 
     case CMBS_EV_DEE_CALL_RELEASECOMPLETE:
-        if(successfullyReleased != 0) successfullyReleased();
         app_OnCallReleaseComplete(pv_AppRef, pv_EventData);
         return TRUE;
 
@@ -4960,7 +4948,6 @@ int app_CallEntity(void *pv_AppRef, E_CMBS_EVENT_ID e_EventID, void *pv_EventDat
         return TRUE;
 
     case CMBS_EV_DEM_CHANNEL_START_RES:
-        if(successfullyConnected != 0) successfullyConnected(u8_callId);
         app_OnChannelStartRsp(pv_EventData);
         return TRUE;
 
