diff --git a/src/opend/dspg/cmbs/test/appcmbs/appsrv.c b/sw_package_host/test/appcmbs/appsrv.c
index f85ac5b..67a8ee2 100644
--- a/src/opend/dspg/cmbs/test/appcmbs/appsrv.c
+++ b/sw_package_host/test/appcmbs/appsrv.c
@@ -73,14 +73,6 @@
 #define APP_SRV_RT_PRINT(format, ...) 							tcx_WriteLog(SRV_MODULE, LOG_LEVEL_REAL_TIME, format,  ##__VA_ARGS__ )
 #define APP_SRV_PRINT_DATA(pu8_Buffer, u16_Length) 				tcx_WriteLogData(SRV_MODULE, LOG_LEVEL_INFO, pu8_Buffer, u16_Length)
 
-static hsSuccessfullyDeregistered successfullyDeregistered;
-
-void initMsgParserDeSub(hsSuccessfullyDeregistered deregisteredClb)
-{
-  if(!deregisteredClb) return;
-  successfullyDeregistered = deregisteredClb;
-}
-
 /* Forward Declaration */
 extern void app_OnHsLocProgress(void *pvAppRefHandle, void *pv_List);
 extern void app_OnHsLocAnswer(void *pvAppRefHandle, void *pv_List);
@@ -247,7 +239,6 @@ void app_OnHandsetRegistered(void *pv_List)
         else if ( st_HsInfo.u8_State == CMBS_HS_REG_STATE_UNREGISTERED )
         {
             APP_SRV_INFO_PRINT("APPSRV-INFO: Handset:%d Unregistered\n", st_HsInfo.u8_Hs);
-            if(successfullyDeregistered != 0) successfullyDeregistered();
 
             List_RemoveHsFromAllLines(st_HsInfo.u8_Hs);
         }
